gravitee-apim-cypress/cypress/integration/bulk/apis-deploy-all.test.ts

```diff
@@ -14,8 +14,8 @@
  * limitations under the License.
  */
 
-import { API_PUBLISHER_USER } from '../../fakers/users/users';
 import { gio } from '../../commands/gravitee.commands';
+import { API_PUBLISHER_USER } from '../../fixtures/fakers/users/users';
```

Comment:
same (path alias)

Reply:
I missed this one, nice catch. Fixed it. :+1: